server/src/server.ts

```diff
@@ -1054,6 +1076,10 @@ function onMessage(msg: p.Message) {
                 workDoneProgress: false,
               }
             : undefined,
+          signatureHelpProvider: {
+            triggerCharacters: ["("],
```

Comment:
Interesting -- I did not know this had its own entire config.

Reply:
Yeah, it's quite elaborate really.